test/suites/device-rest/config_test.go

```diff
@@ -0,0 +1,31 @@
+package test
+
+import (
+	"edgex-snap-testing/test/utils"
+	"testing"
+)
+
+// Deprecated
+func TestEnvConfig(t *testing.T) {
+
+	t.Run("change service port", func(t *testing.T) {
+		t.Cleanup(func() {
+			utils.SnapStop(t, deviceRestService)
+			utils.SnapUnset(t, deviceRestSnap, "env.service.port")
+		})
+
+		const newPort = "56789"
+
+		// make sure the port is available before using it
+		utils.RequirePortAvailable(t, newPort)
+
+		utils.SnapStop(t, deviceRestSnap)
```

Comment:
move this in the begin clean up

Reply:
it would be the same in this case, since there is just one subtest. We will have to refactor and create a setServicePort util function and use it in all relevant suites.